src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
 public @interface Component {
```

Comment:
Do not forget to add @Target to custom annotations.

Reply:
so I need to annotate that my annotation belongs to annotation_type, OK)